keadm/cmd/keadm/app/cmd/util/helminstaller.go

```diff
@@ -18,8 +18,8 @@ import (
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"sigs.k8s.io/yaml"
 
-	keCharts "github.com/kubeedge/kubeedge/build/helm/charts"
 	"github.com/kubeedge/kubeedge/common/constants"
+	keCharts "github.com/kubeedge/kubeedge/helm/charts"
```

Comment:
For import, it's better to use the lowcase?

Reply:
Done.